jdaviz/configs/specviz/tests/test_viewers.py

```diff
@@ -26,6 +26,21 @@ def test_spectrum_viewer_axis_labels(specviz_helper, input_unit, y_axis_label):
     assert (y_axis_label in label)
 
 
+@pytest.mark.xfail(reason="FIXME: Some callback magic needs to happen somewhere.")
```

Comment:
@kecnry , I think this is the more general case you wanted me to fix? My callback-fu isn't strong enough but I added this test here and mark it xfail. If you do follow-up ticket, you can remove the xfail. Thanks.

Reply:
does this currently fail?  We might need to also assert that the actual units on the viewer match, I'm not sure

Reply:
It fails on main and with this patch, yes.